src/js/scripts.js

```diff
@@ -0,0 +1,7 @@
+global.jQuery = require('jquery');
+bootstrap = require('bootstrap');
+mustache = require('mustache');
```

Comment:
Is this being used somewhere? I took a look at your code and didn't found it :/

Reply:
That is correct, those last two lines should have been omitted.

The line 

```js
+bootstrap = require('bootstrap');
```

is used to load all of Bootstrap's plugins to the jQuery object, but none of them ended up being used so that is correct, it should have been removed.

About the last line:

```js
+mustache = require('mustache');
```

Mustache is a JS framework that allows the programmer to create smart templates with a notation much like Angular's, but due to time constraints it ended up never being used.